src/app/tracing/io/io.service.ts

```diff
@@ -29,8 +29,9 @@ export class IOService {
             const jsonData = await getJsonFromFile(file);
             fclData = await this.preprocessData(jsonData);
         } else {
-            const jsonData = await importXlsxFile(file);
+            const { data: jsonData, warnings } = await importXlsxFile(file);
             fclData = await this.preprocessData(jsonData);
+            fclData.importWarnings = [...fclData.importWarnings, ...warnings];
```

Comment:
I would change the issue order to
```
fclData.importWarnings = [...warnings, ...fclData.importWarnings];
```
because the xslx warnings were detected first and might cause other warnings

Reply:
I think this is largely theoretical, since we are initializing an empty fclData in the function one line above, but I didn't want to assume that there woulnd never be any warnings in the object already. Honestly, that might be a bit too defensive, we could very likely just write: 
`fclData.importWarnings = warnings;`

If we do allow the idea that there might already be warnings in the data object, I think it would be correct to append the new warnings at the back of the array.

Reply:
Yes it is currently theoretical, but `preprocessData` converts the external json Structure in the interval format (`fclData`). In my eyes we should not try to make assumptions about the inner workings of that conversion.
Based on the interface we now that fclData might have importWarnings. And these warnings came in after the excel import.